run.py

```diff
@@ -427,6 +440,7 @@ def fit(self):
 	parser.add_argument('-seed',            dest='seed',            default=41504,  type=int,     	help='Seed for randomization')
 
 	parser.add_argument('-restore',         dest='restore',         action='store_true',            help='Restore from the previously saved model')
+	parser.add_argument('-evaluate', 		dest='evaluate', 		action='store_true', 			help='Whether evaluate the model')
```

Comment:
help='Only evaluating a given model, model name must be available in the checkpoints folder'

Reply:
fixed